.github/workflows/integration_test.yml

```diff
@@ -53,3 +53,4 @@ jobs:
         #   battery, connectivity, device_info, package_info, sensors, share, wifi_info_flutter: deprecated.
         #   google_maps_flutter, webview_flutter, url_launcher, network_info_plus: not supported by emulators
         #   tizen_app_control: test available after Flutter 2.5 migration.
+        #   flutter_splash_tizen: non plugin package.
```

Comment:
Packages can also have tests. You could move flutter_splash_tizen to line 48.

Reply:
@swift-kim What comment should I use then? "Runs manually" would fit?

Reply:
You could append it to wearable_rotary and image_select.